### PR TITLE
{data}[intel/2018a and foss/2018a] HDC & PyHDC v0.9.1 {REVIEW}

### PR DESCRIPTION
**Pull request by SimonPinches**
The PR depends upon PR #8850 for MDBM (which is also included in this branch) and PR #8824 (pytest for intel/2018a, now also included in this branch).

### CODE REVIEW

File: easybuild/easyconfigs/p/pytest/pytest-3.8.0-intel-2018a-Python-3.6.4.eb [hunk 1]
```diff
@@ -0,0 +1,56 @@
+easyblock = 'Bundle'
```
Comment (smoors) [inline easybuild/easyconfigs/p/pytest/pytest-3.8.0-intel-2018a-Python-3.6.4.eb]: @SimonPinches according to the first failing travis test, you have to use the `PythonBundle` easyblock for bundles of python packages.

File: easybuild/easyconfigs/p/pytest/pytest-3.8.0-intel-2018a-Python-3.6.4.eb [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+dependencies = [('Python', '3.6.4')]
+
+exts_defaultclass = 'PythonPackage'
+exts_default_options = {
+    'download_dep_fail': True,
+    'use_pip': True,
+}
```
Comment (smoors) [inline easybuild/easyconfigs/p/pytest/pytest-3.8.0-intel-2018a-Python-3.6.4.eb]: with PythonBundle, `exts_defaultclass` and `exts_default_options` can be removed. instead use just this:
```
use_pip = True
```

File: easybuild/easyconfigs/p/pytest/pytest-3.8.0-intel-2018a-Python-3.6.4.eb [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        'checksums': ['a8a07f84e680482eb51e244370aaf2caa6301ef265f37c2bdefb3dd3b663f99d'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': ['bin/pytest'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
+modextrapaths = {'PYTHONPATH': ['lib/python%(pyshortver)s/site-packages']}
```
Comment (smoors) [inline easybuild/easyconfigs/p/pytest/pytest-3.8.0-intel-2018a-Python-3.6.4.eb]: this line can also be removed with PythonBundle.

Comment (SimonPinches) [reply easybuild/easyconfigs/p/pytest/pytest-3.8.0-intel-2018a-Python-3.6.4.eb]: @smoors I finally got around to addressing all your comments in this PR.  Please let me know if you have any other comments.

File: easybuild/easyconfigs/p/pytest/pytest-3.8.0-foss-2018a-Python-3.6.4.eb [hunk 4]
```diff
@@ -11,11 +11,7 @@ toolchain = {'name': 'foss', 'version': '2018a'}
 
 dependencies = [('Python', '3.6.4')]
 
-exts_defaultclass = 'PythonPackage'
-exts_default_options = {
-    'download_dep_fail': True,
-    'use_pip': True,
-}
+use_pip = True
```
Comment (smoors) [inline easybuild/easyconfigs/p/pytest/pytest-3.8.0-foss-2018a-Python-3.6.4.eb]: @SimonPinches the failing tests say that you should add this line as well (this is fairly new):
```
sanity_pip_check = True
```

File: easybuild/easyconfigs/p/pytest/pytest-3.8.0-intel-2018a-Python-3.6.4.eb [hunk 5]
```diff
@@ -11,11 +11,7 @@ toolchain = {'name': 'intel', 'version': '2018a'}
 
 dependencies = [('Python', '3.6.4')]
 
-exts_defaultclass = 'PythonPackage'
-exts_default_options = {
-    'download_dep_fail': True,
-    'use_pip': True,
-}
+use_pip = True
```
Comment (smoors) [inline easybuild/easyconfigs/p/pytest/pytest-3.8.0-intel-2018a-Python-3.6.4.eb]: same here.

File: easybuild/easyconfigs/p/pytest/pytest-3.8.0-foss-2018a-Python-3.6.4.eb (hunk 4 above)
Comment (SimonPinches) [reply easybuild/easyconfigs/p/pytest/pytest-3.8.0-foss-2018a-Python-3.6.4.eb:14]: @smoors, apologies for the delay in commenting, but this is added now.

File: easybuild/easyconfigs/p/pytest/pytest-3.8.0-intel-2018a-Python-3.6.4.eb (hunk 5 above)
Comment (SimonPinches) [reply easybuild/easyconfigs/p/pytest/pytest-3.8.0-intel-2018a-Python-3.6.4.eb:14]: @smoors, also now done.

**Comment by smoors** [comment]
closing-reopening to re-trigger the tests

**Comment by boegel** [comment]
Closing this, since the `foss/2018a` toolchain is now [deprecated](https://docs.easybuild.io/en/latest/Deprecated-easyconfigs.html#deprecated-toolchains)

@SimonPinches If you feel this is still relevant, please consider opening a PR with a more recent toolchain

**Pull request closed without merging**